vllm/model_executor/layers/fused_moe/fused_marlin_moe.py

```diff
@@ -97,16 +128,19 @@ def fused_marlin_moe(
     hidden_states: torch.Tensor,
     w1: torch.Tensor,
     w2: torch.Tensor,
+    w1_scale: torch.Tensor,
+    w2_scale: torch.Tensor,
     gating_output: torch.Tensor,
-    g_idx1: torch.Tensor,
-    g_idx2: torch.Tensor,
-    perm1: torch.Tensor,
-    perm2: torch.Tensor,
     topk_weights: torch.Tensor,
     topk_ids: torch.Tensor,
+    has_zero_point: bool = False,
```

Comment:
Could you just detect has_zero_point from the presence of w1/w2_zeros?

Reply:
I've made it detected in the same way as `has_act_order` is now